packages/sdk/tslint.json

```diff
@@ -1,6 +1,6 @@
 {
   "defaultSeverity": "error",
-  "extends": ["tslint:recommended", "tslint-config-prettier"],
```

Comment:
Why is this removed?

Reply:
![Screenshot from 2021-01-08 16-00-17](https://user-images.githubusercontent.com/69711689/104035920-a9dcde00-51ca-11eb-9fb0-4b9b3bdadaaf.png)

Looks like it changed after you merged this branch with 'master'. I can add `"tslint-config-prettier"` back in if you want? @zie1ony

Reply:
So my bad. Yes, lets bring it back.

Reply:
https://github.com/CasperLabs/clarity/pull/81/commits/6aa79b4d6fe2b5722a2407fd76062bf3cf750c7d - done